src/Sylius/Bundle/ApiBundle/StateProcessor/Processor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\StateProcessor;
+
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ProcessorInterface;
+use Sylius\Bundle\ApiBundle\StateProcessor\Command\CommandAwareInputDataProcessorInterface;
+
+final readonly class Processor implements ProcessorInterface
```

Comment:
The decorated processor is located in the `ApiPlatform\Symfony\Messenger` namespace. What do you think about moving it to `Sylius\Bundle\ApiBundle\ApiPlatform\Messenger`? Or at least, we could rename it to `MessengerProcessor` because the name `Processor` doesn't convey any meaning.